From 942843096e86669492700f49abc2c6489ea9994e Mon Sep 17 00:00:00 2001
From: Aws Ismail <aws.ismail@windriver.com>
Date: Thu, 10 Jan 2013 16:29:05 -0500
Subject: [PATCH] Set the PID Lock path for samhain.pid

The explicit path for samhain.pid inorder
for samhain to work properly after it initial
database build.

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Aws Ismail <aws.ismail@windriver.com>
---
 samhainrc.linux | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/samhainrc.linux b/samhainrc.linux
index ff6ddf1..d4b71e9 100644
--- a/samhainrc.linux
+++ b/samhainrc.linux
@@ -621,7 +621,7 @@ SetFileCheckTime = 86400
 
 ## Path to the PID file
 #
-# SetLockfilePath = (default: compiled-in)
+SetLockfilePath = /run/samhain.pid
 
 
 ## The digest/checksum/hash algorithm
-- 
2.34.1

